controller/persistence/transit_router_store.go

```diff
@@ -65,6 +66,11 @@ func (entity *TransitRouter) LoadValues(store boltz.CrudStore, bucket *boltz.Typ
 	_, err := store.GetParentStore().BaseLoadOneById(bucket.Tx(), entity.Id, &entity.Router)
 	bucket.SetError(err)
 
+	if bucket.Bucket == nil {
```

Comment:
Stores that extend need to handle situations where the  child bucket is not defined. i.e. skip or provide info to the handler/REST level.